app/controllers/new_admin/questionnaires_controller.rb

```diff
@@ -67,6 +68,16 @@ def questionnaire_params
                                             questions_attributes: %i[title kind raw_answer_options _destroy id])
     end
 
+    def questions_to_be_destroyed_ids
+      return nil unless questionnaire_params[:questions_attributes]
```

Comment:
This should suffice:
```suggestion
      return unless questionnaire_params[:questions_attributes]
```

Reply:
done